src/components/Limit.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+	<div class="querybuilder__limit">
```

Comment:
this classname isn't exactly ideal BEM, but we have this problem in other places as well. We'll need to do a general sweep of class names to align all of them. But not in this PR.

Reply:
oh, i was taking other elements as guide. O: yes, makes sense to change them all at once if this is not the best way